### ISSUE TITLE
assert statement fails in exponentiated gradient reduction (sklearn compatible) notebook

### ISSUE THREAD

**Issue by nrkarthikeyan** [Trusted-AI/AIF360#225]
The assert  statement

```assert egr_aoe_race<lr_aoe_race```

used in  the notebook example demo_exponentiated_gradient_reduction_sklearn.ipynb seems to  fail  when run in travis while it passes locally. Needs further investigation.

For now, we commented this to merge PR #215, but we can incorporate this  test back again after fixing this issue.
